### DISCUSSION TITLE
How to add nikola some site analytics(goatcounter)?

### DISCUSSION THREAD

**Discussion by UnbalancedSkunk** [getnikola/nikola#3778] [Q&A] [answered]
İ would like to add goatcounter to nikola. İ used base.tmpl file before </body> but failed.  My theme bnw

**Comment by UnbalancedSkunk** [comment]
i noticed that problem is about ublock origin

**Reply by Kwpolska** [reply, marked as answer]
`base.tmpl` is one option, there might be others depending on the theme. Check the resultant HTML, and make sure to disable ad blockers and other privacy protection tools.
